.ci/scripts/install-test-dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+#
+# Install the dependencies using in tests.
+#
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+mv ./kubectl /usr/local/bin/kubectl
```

Comment:
Can we use `sudo` in a worker? Otherwise this will fail

Reply:
no, we can not, because of that we install everything in the Jenkins workspace job